src/api/server/server.js

```diff
@@ -63,28 +62,43 @@ function getServerInfoAsync(
   });
 }
 
-function getFee(): number {
-  return common.dropsToXrp(this.remote.createTransaction()._computeFee());
+function getFee(): ?number {
+  if (!this.remote._servers.length) {
+    throw new common.errors.RippledNetworkError('No servers available.');
+  }
+  const fee = this.remote.createTransaction()._computeFee();
+  if (typeof fee !== 'string') {
```

Comment:
Isn't the only non-string return value `undefined`? If so, it's better to test for undefined than to test the type.